src/status_im/utils/mixpanel_events.edn

```diff
@@ -21,6 +21,14 @@
  {:label      "Logout"
   :trigger    [:navigate-to :accounts]}
 
+ ; Push notification settings
+
+ {:label      "Granted push notifications"
```

Comment:
@chadyj is it ok for you?

Reply:
Wow cool! @lukaszfryc can you test it out?

Reply:
yeah, will do